anaUltraScurve.py

```diff
@@ -54,13 +54,12 @@
         pass
     pass
 
-buildHome = os.environ.get('BUILD_HOME')
-
+projectHome = os.environ.get('GEM_PLOTTING_PROJECT')
```

Comment:
Checks made using `envCheck()` in related scripts:

- `anaUltraScurve.py`
- `anaUltraThreshold.py`
- `mapping/buildMapFiles.py`